compliance/usg/cis-benchmarks.rst

```diff
@@ -1,28 +1,30 @@
 CIS benchmarks and profiles
```

Comment:
From what I can tell you are [correct](https://www.cisecurity.org/cis-benchmarks):
```suggestion
CIS Benchmarks and profiles
```